src/stories/Quiz/Component/index.stories.tsx

```diff
@@ -48,8 +48,19 @@ addComponentStoryDescription(
 
 const cioJsClient = new ConstructorIOClient({ apiKey });
 
+// The following block is to remove unrelated modules from Storybook's "Controls" panel
+// This should not be done in an actual application
+// @ts-ignore
+delete cioJsClient.autocomplete;
+// @ts-ignore
+delete cioJsClient.search;
+// @ts-ignore
+delete cioJsClient.browse;
+// @ts-ignore
+delete cioJsClient.recommendations;
+
 export const ProvideCIOClientInstance = ComponentTemplate.bind({});
-ProvideCIOClientInstance.args = { quizId, cioJsClient, resultsPageOptions };
+ProvideCIOClientInstance.args = { quizId, resultsPageOptions, cioJsClient };
```

Comment:
I'm not sure if moving the args here works 🤔 

Might need to make adjustments to the [stringifyWithDefaults](https://github.com/Constructor-io/constructorio-ui-quizzes/blob/d37e3f90ab1c4a13e2faebe0d287daf1de3607a9/src/utils.tsx#L64) util function.